scripts/check-size-firmware.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+DTB=$1
+IMAGE_FILE_SIZE=$2
+PARTITION_NODES="/ahb/spi/flash/partitions /spi/flash/partitions"
+FDGET=fdtget
```

Comment:
This should be removed, and defined as `FDTGET:=fdtget` in `rules.mk`.

Reply:
rules.mk is for Makefiles.
I will go backwards here and just hardcode "fdtget" it in the lines below.

Reply:
Will the fdtget from your path be "in path" here, or will I have to use an explicit path?